practice/00-HTML-CSS-basics/botom.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>Form</title>
+</head>
+<body>
+<div>
+	<h1>sign up</h1>
+	<form class="contact_form">
+		<ul>
+				<li><label for="name">Name: </label><input type="text" placeholder="Michael" required></li>
+                <li><label for="name">Last Name: </label><input type="text" placeholder="Jackson" required></li>
```

Comment:
The id's are unique identifiers so they must be unique. Use another id here like last_name